app/views/coach/_coach.html.haml

```diff
@@ -1,20 +0,0 @@
-.stripe.reverse
-  .row
-    .large-12.columns
-      %ul.small-block-grid-1.medium-block-grid-2.large-block-grid-3
-        - coaches.each do |coach|
-          %li.coach
-            .row
-              .small-2.columns
-                =link_to twitter_url_for(coach.twitter) do
-                  =image_tag coach.avatar(96), class: 'th radius', title: coach.full_name
-
-              .small-10.columns
-                =link_to twitter_url_for(coach.twitter) do
-                  = coach.full_name
-                %p= coach.about_you
-                - if coach.skills.any?
-                  %p 
-                    skills: 
-                    - coach.skills.each do |skill|
-                      = link_to skill.name, skill_url(skill.name), class: "skill"
```

Comment:
Rendering `coach/coach` and passing a collection of coaches seemed confusing to me, I renamed & broke out this file to make its purpose more clear.